{{cookiecutter.github_repository}}/{{cookiecutter.main_module}}/graphql/users/resolvers.py

```diff
@@ -0,0 +1,5 @@
+from django.contrib.auth import get_user_model
+
+
+def resolve_users(info):
```

Comment:
the method caller is also named resolve users. In order to avoid any naming resolution in future, will it make sense to rename this resolver to something else? 

or for now, not make this resolvers file, but keep it inside the query.

Let me know what your thoughts are.

Reply:
+1 for renaming